app/controllers/categories_controller.rb

```diff
@@ -0,0 +1,72 @@
+class CategoriesController < ApplicationController
+  before_action :set_category, only: %i[show edit update destroy]
+
+  # GET /categories or /categories.json
+  def index
+    @categories = Category.where(user: current_user)
+  end
+
+  # GET /categories/1 or /categories/1.json
+  # def show
+  #   @transactions1s = @category.transactions1
+  # end
+
```

Comment:
- [ ] Kindly remove commented code on your online repository

Reply:
This issue is well resolved! ✔️